dev.restfox.Restfox.yaml

```diff
@@ -0,0 +1,40 @@
+app-id: dev.restfox.Restfox
+base: org.electronjs.Electron2.BaseApp
+base-version: "24.08"
+command: restfox
+finish-args:
+  - --device=dri
+  - --filesystem=home
```

Comment:
why is home needed?

Reply:
Files/Collections need to be saved/read from the filesystem. It's difficult to lock down users to certain locations, so to maintain the same experience with the original release, this is needed at the moment.

This application behaves similar to Postman and Insomnia.